mbuild/compound.py

```diff
@@ -999,7 +999,7 @@ def center(self):
             The cartesian center of the Compound based on its Particles
 
         """
-        if self.xyz.any():
+        if len(self.xyz) != 0:
```

Comment:
These are pretty different checks. What prompted this change?

Reply:
I'm still in the middle of working on this PR. This was just a temporary change (that will be fixed when #386 is merged).